src/backend/api/Fusion.Resources.Api/Deployment/k8s/deployment-pr-env.yml

```diff
@@ -0,0 +1,129 @@
+﻿apiVersion: v1
+kind: Secret
+metadata:
+  name: env-secrets
```

Comment:
How will this act when there are multiple environments? 
Maybe the pr pipeline should rather just ensure that there exists one secret that has an up to date secret? I guess `apply -f ..` here will just update the existing key?
Maybe just remove the labels and let this be updated for each new pr deploy.

This would just change the prNumber label on the same secret, and if it was to be removed by a cleanup, all existing pods would fail.

Reply:
Good catch, will remove the label and make it a common secret.